src/signals/incident/components/form/MapSelectors/Asset/Selector/DetailPanel/DetailPanel.tsx

```diff
@@ -54,17 +60,39 @@ const nearbyLegendItem = {
   typeValue: NEARBY_TYPE,
 }
 
+type Point = {
+  type: 'Point'
+  coordinates: LatLngTuple
+}
+
+type Properties = {
+  category: {
+    name: string
+  }
+  created_at: string
+}
+
+interface SelectionIncident {
```

Comment:
Why is Properties a type and interface a SelectionIncident?

Reply:
I will change SelectionIncident to a type to be more consistent